indico/modules/events/models/persons.py

```diff
@@ -242,6 +242,17 @@ def link_user_by_email(cls, user):
                 db.session.delete(event_person)
         db.session.flush()
 
+    def sync_user(self):
```

Comment:
Not sure about the further uses of `sync_user` but would it make sense to trigger the `person-updated` signal?

Reply:
When cloning no, but when doing it in an existing event it probably makes sense. So I'll probably add a kwarg for it and default to true..